### ISSUE TITLE
Can not compile on Ubuntu

### ISSUE THREAD

**Issue by sina-arkel** [KDAB/GammaRay#163]
Hello, I have downloaded 2.3.0 and followed `Install.txt`;

Building on Unix with gcc or clang:
% mkdir build
% cd build
% cmake ..
% make
% make install

like it says, i have installed dependencies but on the make step, it stops compiling and gives that error,

[  3%] Building CXX object common/CMakeFiles/gammaray_common.dir/gammaray_common_automoc.cpp.o
Linking CXX shared library ../lib/libgammaray_common-qt5.2-x86_64.so
/usr/bin/ld: CMakeFiles/gammaray_common.dir/__/3rdparty/lz4/lz4.c.o: Relocations in generic ELF (EM: 40)
CMakeFiles/gammaray_common.dir/__/3rdparty/lz4/lz4.c.o: error adding symbols: File in wrong format
collect2: error: ld returned 1 exit status
make[2]: *** [lib/libgammaray_common-qt5.2-x86_64.so.2.3.0] Error 1
make[1]: *** [common/CMakeFiles/gammaray_common.dir/all] Error 2
make: *** [all] Error 2

I am not sure if I am doing something wrong or is there an issue or not. 

Thanks, 
Sina

**Comment by krf** [comment]
Did you try cleaning the build directory and starting from scratch? Are you trying to cross-compile GammaRay?

Can't really explain this issue...

**Comment by sina-arkel** [comment]
yes I have tried to build from scratch. Actually my intend is use GammaRay for my Raspberry Pi but I just tried to build it for my host machine first which is Ubuntu. 

But I managed to compile it for my host machine just now, cmake was looking for qmake, and since I am using QtCreator, qmake was missing in my $PATH. I just added and it did compiled.

So I will close it for now, but probably I'll come back for cross compiling :),

Thanks,
Sina
